src/sentry/migrations/0021_auto_20191202_1716.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2019-12-02 17:16
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Adding indexes to large tables. These indexes should be created concurrently,
+    #   unfortunately we can't run migrations outside of a transaction until Django
+    #   1.10. So until then these should be run manually.
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+
+    dependencies = [
+        ('sentry', '0020_auto_20191125_1420'),
+    ]
+
+    operations = [
```

Comment:
At this point, can we verify that this migration does nothing? It seems like it would, so this is probably most likely fixing internal state.

Reply:
Running `sentry django sqlmigrate sentry 0021` returns

```
BEGIN;
--
-- Rename model LatestRelease to LatestRepoReleaseEnvironment
--

COMMIT;
```

So yes, it's a no-op. :-)